### ISSUE TITLE
Change Color of Sunday and Saturday Columns

### ISSUE THREAD

**Issue by SeifBenAbda** [mytooyo/board_datetime_picker#27]
![Screenshot 2024-04-28 105829](https://github.com/mytooyo/board_datetime_picker/assets/42813983/0b42e815-babe-45aa-9454-8519dd4cb945)

Is there a way to Change the Color of both Columns of Sunday and Saturday (Blue and Red) to a Custom Color

**Comment by mytooyo** [comment]
> ![Screenshot 2024-04-28 105829](https://github.com/mytooyo/board_datetime_picker/assets/42813983/0b42e815-babe-45aa-9454-8519dd4cb945)
> 
> 
> 
> Is there a way to Change the Color of both Columns of Sunday and Saturday (Blue and Red) to a Custom Color

It does not appear to be possible to change it at this time.
Please wait until it is supported to specify it.

**Comment by mytooyo** [comment]
@SeifBenAbda 

Thanks for the request.
I have addressed the following in the 1.6.4 version.

```dart
options: BoardDateTimeOptions(
  weekend: BoardPickerWeekendOptions(
    sundayColor: Colors.yellow,
    saturdayColor: Colors.red,
  ),
),
```

**Comment by SeifBenAbda** [comment]
Thank you for the update @mytooyo
